rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/shared/RMConfig.java

```diff
@@ -79,7 +79,7 @@
     /** stat history refresh rate in millis */
     public static final String STATISTICS_REFRESH_TIME = "rm.client.stats.time";
 
-    private static final String d_STATISTICS_REFRESH_TIME = "5000";
+    private static final String d_STATISTICS_REFRESH_TIME = "3000";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)